fvm/scriptEnv.go

```diff
@@ -186,6 +186,10 @@ func (e *ScriptEnv) ValueExists(owner, key []byte) (exists bool, err error) {
 	return len(v) > 0, nil
 }
 
+func (e *ScriptEnv) AccountExists(address common.Address) (exists bool, err error) {
```

Comment:
I don't see this being used anywhere, or did I miss something.

Reply:
not used anywhere yet, but I need it for the next PR.